web-ng/src/app/services/router/router.service.ts

```diff
@@ -33,6 +33,10 @@ export class NavigationService {
   private static readonly OBSERVATION_ID_FRAGMENT_PARAM = 'o';
   static readonly LAYER_ID_NEW = 'new';
   static readonly OBSERVATION_ID_NEW = 'new';
+  static readonly PROJECT_ID_NEW = 'new';
+  static readonly PROJECT_ID = 'projectId';
+  static readonly DEFAULT_PROJECT_ROUTE = 'project';
```

Comment:
Why is this the "default" project route? Shouldn't it be something like the "project URL base path segment"? (`PROJECT_SEGMENT`?)